docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -502,49 +949,249 @@ public class PersonRepository implements PanacheRepositoryBase<Person,Integer> {
     //...
 }
 ----
+====
+
+
+
+== Defining entities in external projects or JAR files
```

Comment:
This has been substantially rewritten and shortened upstream.

Reply:
Do you mean in the Hibernate ORM with Panache guide? I expanded this section because I felt the very brief explanation in that guide was insufficient.

Reply:
Yes, in the current Quarkus repo.

Reply:
I agree with Emily here, we needed more information for our customers. Her expansion seems perfect.

Reply:
What I mean is that upstream this section got rewritten entirely, check it out: https://quarkus.io/guides/hibernate-orm-panache#defining-entities-in-external-projects-or-jars

So I think your rewrite is no longer relevant. Sorry, this often happens with Quarkus because upstream moves fast.